recipes/cdutil/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cdutil" %}
+{% set version = "2.8.2" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/UV-CDAT/cdutil/archive/{{ version }}.tar.gz
+    sha256: a9eb7e4805a9b9f1acc46b84e2e241260e9690ca76cba5331eba2631ec9403b1
+    
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - genutil
+  run:
+    - python
+    - cdms2
+    - numpy x.x
+    - udunits2
+    - genutil
+
+test:
```

Comment:
tried it but in a virgin env we first ask user yes/no for some env to be set. I do not know how to pass an env variable to meta.yaml (google search suggest activate.d but i do not want to set the env variable to any default value for the distributed version)